etc/workflows/failedRecording.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>failedrecording</id>
+  <title>Recording failed</title>
+  <tags>
+  <tag>editor</tag>
+  <tag>archive</tag>
+  </tags>
+  <description>
+    Publish a placeholder video for corrupt videos.
+    Delete and cleanup old snapshots, but keep the metadata.
+  <description/>
+  <configuration_panel/>
+
+  <operations>
+
+    <!-- Resolve the comment for cutting -->
+    <operation
+      id="comment"
+      description="Resolve the cutting flag">
+      <configurations>
+        <configuration key="description">Recording has not been cut yet.</configuration>
+        <configuration key="reason">EVENTS.EVENTS.DETAILS.COMMENTS.REASONS.CUTTING</configuration>
+        <configuration key="action">resolve</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Untag elements for deletion -->
+    <operation
+      id="tag"
+      max-attempts="2"
+      fail-on-error="true"
+      exception-handler-workflow="error"
+      description="Tagging media package elements for deletion">
+      <configurations>
+        <configuration key="source-flavors">*/trimmed,*/source</configuration>
+        <configuration key="target-tags">-atom,-archive</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Copy placeholder -failed- Video for publication -->
+    <operation
```

Comment:
Once you convinced @darolfes to generalize his WOH (see https://github.com/opencast/opencast/pull/710), you can just add the video without mis-using an encoding profile to copy files ;-)

Note that it would be an option to use WOH image-to-video in combination with an failed.jpg instead of failed.mp4... but just mentioning this.